src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=7.1.3",
         "ext-xml": "*",
         "symfony/cache": "^4.4|^5.0",
-        "symfony/config": "^4.3.4|^5.0",
+        "symfony/config": "^4.4.11|~5.0.11|^5.1.3",
```

Comment:
Version 5.1.0 is still allowed here. 

Could you elaborate why do you update this line?

Reply:
I've explained this in an earlier comment, but that got lost after a push, apparently.

> I've bumped this constraint because I need the fix from #37526 for the low-deps build.